pyproject.toml

```
@@ -62,6 +62,8 @@ dev = [
   "pandas",
   "awkward-pandas"
 ]
+http = []
```

I think this is what fsspec needs for HTTP access.

```suggestion
http = ["aiohttp"]
```